### PR TITLE
Add Prepd app to showcase

### PR DESCRIPTION
**Pull request by marcelkalveram**
This app is written entirely in React Native, and available on the iOS and Google Play app stores.

Blogpost: https://logbook.hanno.co/prepd-pros-cons-react-native/
Case study: https://hanno.co/work/prepd/

### CODE REVIEW

**Comment by hramos** [comment]
We can't accept this PR as is - the entry cannot be added to the pinned array. Please refer to the guidelines in the file.

**Comment by marcelkalveram** [comment]
Thanks for reviewing this @hramos. 

I can't seem to figure out what's wrong with the commit though. I have doubled-checked the format and the guidelines and checked if all the links are valid. 

Could you provide some more detail as to what's wrong/missing?

**Comment by ide** [comment]
He's saying to move your entry from pinned to featured because
> The pinned list is reserved for a small list of hand-picked apps.

**Comment by marcelkalveram** [comment]
Oops, my bad. And thanks for clarifying @ide. I've moved the app to the featured array.

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -369,6 +369,15 @@ var featured = [
...
     infoTitle: 'Social network that connects entrepreneurs to fellow entrepreneurs, consumers, investors and experts',
   },
+  {
+    name: 'Prepd',
+    icon: 'http://is4.mzstatic.com/image/thumb/Purple111/v4/03/56/95/035695ab-77dc-4f3b-0961-3263e32e3494/source/175x175bb.jpg',
+    linkAppStore: 'https://itunes.apple.com/us/app/get-prepd/id1125687122?mt=8',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.prepd&hl=en',
+    infoLink: 'https://logbook.hanno.co/prepd-pros-cons-react-native/',
+    infoTitle: 'How we built Prepd for iOS and Android using React Native',
+    defaultLink: 'https://itunes.apple.com/us/app/get-prepd/id1125687122?mt=8',
```
Comment (hramos) [inline website/src/react-native/showcase.js]: `defaultLink` is not necessary here, you can delete this line.

**Comment by marcelkalveram** [comment]
Done, thanks!

**Pull request closed without merging**